### PR TITLE
RHDEVDOCS-4420 - Add a unique cluster name or identifier to the log record

### PR DESCRIPTION
**Pull request by libander**
Version(s): 4.10+

Issue: https://issues.redhat.com/browse/RHDEVDOCS-4420

Link to docs preview: n/a, snippet. 

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: snippets/logging-get-clusterid-snip.adoc [hunk 1]
```diff
@@ -0,0 +1,12 @@
+// Text snippet included in the following modules and assemblies:
+//
+
+:_content-type: SNIPPET
+
+Logs from any source contain a clusterID, the unique identifier of the cluster in which the Operator is deployed.
```
Comment (kabirbhartiRH) [inline snippets/logging-get-clusterid-snip.adoc:6]: Do you think this sounds better? 
_contains a field 'openshift.cluster_id', the unique identifier of the cluster in which the Operator is deployed._

**Review by rolfedh — APPROVED** [review]
LGTM

**Pull request merged**